frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/BuilderAndDesignContent.tsx

```diff
@@ -15,69 +29,158 @@ import BuilderAndDesignPlaceholder from './BuilderAndDesignPlaceholder'
 import { BuilderFields } from './BuilderFields'
 import { useBuilderFields } from './useBuilderFields'
 
+interface FormBuilderProps {
+  placeholderProps: DndPlaceholderProps
+}
+
+const FormBuilder = ({ placeholderProps }: FormBuilderProps): JSX.Element => {
+  const { builderFields } = useBuilderFields()
+  const { handleBuilderClick } = useCreatePageSidebar()
+  const setEditEndPage = useBuilderAndDesignStore(setToEditEndPageSelector)
+
+  return (
+    <Flex
+      m={{ base: 0, md: '2rem' }}
+      mb={0}
+      flex={1}
+      bg={{ base: 'secondary.100', md: 'primary.100' }}
+      p={{ base: '1.5rem', md: '2.5rem' }}
+      justify="center"
+      overflow="auto"
+    >
+      <Flex
+        h="fit-content"
+        bg="white"
+        p={{ base: 0, md: '2.5rem' }}
+        maxW="57rem"
+        w="100%"
+        flexDir="column"
+      >
+        <Droppable droppableId={FIELD_LIST_DROP_ID}>
+          {(provided, snapshot) =>
+            builderFields?.length ? (
+              <Box
+                pos="relative"
+                ref={provided.innerRef}
+                {...provided.droppableProps}
+              >
+                <BuilderFields
+                  fields={builderFields}
+                  isDraggingOver={snapshot.isDraggingOver}
+                />
+                {provided.placeholder}
+                <BuilderAndDesignPlaceholder
+                  placeholderProps={placeholderProps}
+                  isDraggingOver={snapshot.isDraggingOver}
+                />
+              </Box>
+            ) : (
+              <EmptyFormPlaceholder
+                ref={provided.innerRef}
+                {...provided.droppableProps}
+                isDraggingOver={snapshot.isDraggingOver}
+                onClick={handleBuilderClick}
+              />
+            )
+          }
+        </Droppable>
+        <Button
+          py="1.5rem"
+          mt="1.5rem"
+          variant="outline"
+          colorScheme="secondary"
+          onClick={() => {
+            setEditEndPage()
+            handleBuilderClick()
+          }}
+        >
+          <Text textStyle="subhead-2">Thank you</Text>
+        </Button>
+      </Flex>
+    </Flex>
+  )
+}
+
+const EndPageView = (): JSX.Element => {
+  const { data: form } = useAdminForm()
+  const endPageData = useEndPageBuilderStore(endPageDataSelector)
+
+  return (
+    <Flex
+      m={{ base: 0, md: '2rem' }}
+      mb={0}
+      flex={1}
+      bg="white"
+      p={{ base: '1.5rem', md: 0 }}
+      justify="center"
+      overflow="auto"
+    >
+      <Stack w="100%">
+        <Flex justifyContent="center" pt="1rem" pb="0.5rem">
+          <Image src={form?.admin?.agency?.logo ?? ''} h="4rem" />
```

Comment:
nit: do we need the empty string `''` default? `src` accepts `undefined`

Reply:
Oh then we don't need `''` default! Removed it